Gemfile.lock

```diff
@@ -559,6 +568,7 @@ DEPENDENCIES
   sass-rails
   scrypt
   selenium-webdriver
+  shoulda-matchers (= 4.0.0.rc1)
```

Comment:
I like to be conservative with adding dependencies, as each one contributes to complexity, cost of maintenance, performance, etc. I usually will ask things like: Are we using this gem right now? If so, which features? Can we get away without it or will it save us a lot of work in the future? Can we remove some other gem if we are going to add this one?